components/Alert/AlertNotification.tsx

```diff
@@ -0,0 +1,38 @@
+import { Alert as AlertDefault, AlertTitle, AlertDescription } from './Alert';
+import { CheckCircle, XCircle, Info, AlertTriangle } from 'lucide-react';
+
+interface AlertProps {
+  variant?: 'success' | 'error' | 'warning';
```

Comment:
Make this list into an ENUM. Then, this is a strict list of our accepted alerts. Also, require variants. Set a default for variant, and the prop wont complain. Default is just is a normal variant.